server/graphql/v2/interface/Transaction.js

```diff
@@ -252,5 +261,20 @@ export const TransactionFields = () => {
           : null;
       },
     },
+    isRejected: {
+      type: GraphQLBoolean,
+      async resolve(transaction, _, req) {
+        if (transaction.OrderId) {
+          const order = await req.loaders.Order.byId.load(transaction.OrderId);
+          if (order.status === orderStatus.REJECTED) {
+            return true;
+          } else {
+            return false;
+          }
```

Comment:
**[:ice_cream: Details & personal preferences]** 

```suggestion
          return order.status === orderStatus.REJECTED;
```

Reply:
Omg of course 🤦‍♀️